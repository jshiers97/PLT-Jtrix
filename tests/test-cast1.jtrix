int main(){
int i;
float f;
i = 3;
f = int_to_f(i);
print(i);
printf(f);
f = int_to_f(i) + f;
printf(f);
} 
