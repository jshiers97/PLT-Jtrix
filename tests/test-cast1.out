3
3.000
6.000
